pyne/serpent.py

```diff
@@ -261,6 +267,7 @@ def parse_dep(depfile, write_py=False, make_mats=True):
     # Add materials
     footer = ""
     if make_mats:
+        quicksave_f = copy.deepcopy(f)
```

Comment:
I'm sorry, I don't think I understand what you mean.  I made a deep copy of f at that line to save a version that does not have any added materials.  Then, if the original mat_gen_line doesn't work, instead of finding and inserting the correct characters, I start with quicksave_f.  What would be the new variable?

Reply:
see my example below, 

basically you can keep the `f` variable untouched and just make one with the header and the footer: `f_full = header + f + footer`

Reply:
instead of `f = header + f + footer`

Reply:
Ah, because the mat_gen_line is only in the footer, and f_full is a new variable, so you don't have to worry about one changing the other.  That makes sense, thanks!